docs/recipes.rst

```diff
@@ -930,54 +385,24 @@ Measure temperature with an ADC
 .. IMAGE TBD
 
 Wire a TMP36 temperature sensor to the first channel of an :class:`MCP3008`
-analog to digital converter::
-
-    from gpiozero import MCP3008
-    from time import sleep
-
-    def convert_temp(gen):
-        for value in gen:
-            yield (value * 3.3 - 0.5) * 100
+analog to digital converter:
 
-    adc = MCP3008(channel=0)
-
-    for temp in convert_temp(adc.values):
-        print('The temperature is', temp, 'C')
-        sleep(1)
+.. literalinclude:: examples/thermometer.py
 
 
 Full color LED controlled by 3 potentiometers
 =============================================
 
 Wire up three potentiometers (for red, green and blue) and use each of their
-values to make up the colour of the LED::
-
-    from gpiozero import RGBLED, MCP3008
+values to make up the colour of the LED:
 
-    led = RGBLED(red=2, green=3, blue=4)
-    red_pot = MCP3008(channel=0)
-    green_pot = MCP3008(channel=1)
-    blue_pot = MCP3008(channel=2)
-
-    while True:
-        led.red = red_pot.value
-        led.green = green_pot.value
-        led.blue = blue_pot.value
+.. literalinclude:: examples/rgbled_pot_1.py
 
 Alternatively, the following example is identical, but uses the
-:attr:`~SourceMixin.source` property rather than a :keyword:`while` loop::
-
-    from gpiozero import RGBLED, MCP3008
-    from signal import pause
+:attr:`~SourceMixin.source` property rather than a :keyword:`while` loop:
 
-    led = RGBLED(2, 3, 4)
-    red_pot = MCP3008(0)
-    green_pot = MCP3008(1)
-    blue_pot = MCP3008(2)
-
-    led.source = zip(red_pot.values, green_pot.values, blue_pot.values)
-
-    pause()
+.. literalinclude:: examples/rgbled_pot_2.py
+    :emphasize-lines: 8
```

Comment:
Offline build's using the default theme - unfortunately I think it's unaware of the emphasis styles. But then the offline build has to remain compatible with whatever downlevel version of sphinx is supported in Raspbian at the moment (debuild ensures that venv's aren't used during package build so you can try and use a later sphinx, but it won't work - you'll wind up on the raspbian installed version)